### ISSUE TITLE
doi -> full citation in references.md

### ISSUE THREAD

**Issue by ypar** [greenelab/cimr-d#49]
Currently, cimr-d recommends data submitters to indicate doi(s) of data or publication(s) to cite when specific submitted data is used. This is then stored in the catalog.txt. These doi should lead to a full citation (.ris, .bib or such likes) in cimr-d references.md.

**Comment by cgreene** [comment]
The manubot citation infrastructure could be used to fill this in. Are you thinking that the full citation would go somewhere like the selected space in this screenshot?
![Screen Shot 2019-09-06 at 9 09 01 AM](https://user-images.githubusercontent.com/542643/64430310-06cffd80-d086-11e9-9954-12832df4e8f5.png)

**Comment by ypar** [comment]
Yes. doi on the README.md of the processed dir with a link to references.md with a full citation will be best. To fit each file info per line, timestamps may need shortening (e.g. no h m s). I will give it a bit more thought.

**Comment by ypar** [comment]
Putting this as a reminder to myself. I think the useful format for the README.md in the processed dir would be, e.g.

* data_type
    * [context such as disease or tissue name](aws s3 hyperlink): file_size, [doi](citation file, article link or specific line in cimr-d references.md), [metadata](processed yaml path), updated on ZZZZ

It will be some work trying to pretty-it-up, but it will be more informative than listing file names that tend to be very study-specific.
